docs/types.rst

```diff
@@ -857,14 +859,35 @@ or create a new memory array and copy every element.
         }
     }
 
-.. index:: ! array;literals, !inline;arrays
+.. index:: ! array;literals, ! inline;arrays
 
-Array Literals / Inline Arrays
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Array Literals
+^^^^^^^^^^^^^^
 
-Array literals are arrays that are written as an expression and are not
+Array literals (also called inline arrays) are arrays that are written as an expression and are not
 assigned to a variable right away.
 
+You create a statically sized memory array literal using syntax such as
+``string[] memory myarray = new string[](4);``.
```

Comment:
There is no array literal in this example.

Reply:
The example below is correct, only this one does not fit this section.

Reply:
This came from the FAQ point, I think the inline example is enough anyway, so let's just remove the longer example.

Reply:
This example here is wrong. The one you removed is correct, and I think it is a good example.